source/views/stoprint/components/error.js

```diff
@@ -27,13 +35,28 @@ export class StoPrintErrorView extends React.PureComponent<Props> {
 		}
 	}
 
+	_refresh = () => {
+		this.setState(() => ({refreshing: true}))
+		this.props.refresh()
+		this.setState(() => ({refreshing: false}))
```

Comment:
You should add the delay things that we use on the list views here, as well, please.